firmware/projects/FrontController/platforms/stm32f767/cubemx/README

```diff
@@ -0,0 +1,5 @@
+All files generated by CubeMX will be placed here.
+
+The .ioc file is the only one tracked under version control. It contains all the necessary information to generate the other files.
+
+Before building the project, you are expected to open the .ioc and click 'Generate Code' locally.
```

Comment:
Update comment

Reply:
I will make this an issue since it needs to be applied to all projects